Xamarin.Essentials/DeviceInfo/DeviceInfo.uwp.cs

```diff
@@ -68,5 +75,23 @@ static DeviceType GetDeviceType()
 
             return DeviceType.Physical;
         }
+
+        static async Task<List<StorageInfo>> PlatformGetStorageInformation()
+        {
+            var storageInfos = new List<StorageInfo>();
+            var folders = await KnownFolders.RemovableDevices.GetFoldersAsync();
+            foreach (var folder in folders)
+            {
+                var folderProps = await folder.Properties.RetrievePropertiesAsync(properties);
+                var fCapacity = (ulong)folderProps[capacity];
+                var fFree = (ulong)folderProps[freeSpace];
+                storageInfos.Add(new StorageInfo(fCapacity, fFree, fCapacity - fFree, StorageType.External));
+            }
+            var localProps = await ApplicationData.Current.LocalFolder.Properties.RetrievePropertiesAsync(properties);
+            var localCapacity = (ulong)localProps[capacity];
+            var localFree = (ulong)localProps[freeSpace];
+            storageInfos.Add(new StorageInfo(localCapacity, localFree, localCapacity - localFree, StorageType.External));
```

Comment:
Should this be internal?

```suggestion
            storageInfos.Add(new StorageInfo(localCapacity, localFree, localCapacity - localFree, StorageType.Internal));
```